### PR TITLE
20201113-nanodiam-1.0.0.md

### PR DESCRIPTION
**Pull request by adrianm3**
|||
|---|---|
|Author(s)|Adrian Manjarrez|
|Status|Closed|
|Phase| 1|
|Open Date|11-19-2020|
|Close Date|12-03-2020|

### CODE REVIEW

**Comment by dadamsncsa** [comment]
Are there any more folks we can add as reviews or contributors before opening the review period?

**Comment by dadamsncsa** [comment]
test comment for slack

**Comment by dadamsncsa** [comment]
Are there any more folks we can add as reviews or contributors before opening the review period?

**Comment by elifleaf** [comment]
Other reviewers could be Lili, Sam, and Aagam.  I'm not sure how to tell who (or if) anyone has been formally assigned yet.

**Comment by dadamsncsa** [comment]
@elifleaf @adrianm3 @greghbauer @AagamShah97 @tawfick 
The review RFC period for this project is now open, refer to the phase 1 review guide here:
https://github.com/nanoMFG/community/blob/master/governance/planning-review.md
 for phase 1 review criterial.  Also, take advantage of the line comment capabilities.  The document to be reviewed can most easily be access by visiting the "Files Changed" tab above.  I pinged Lili Cai on slack to setup a github account so she can be added as a reviewer.

**Review by dadamsncsa — COMMENTED** [review]
# Phase 1 Review
Hello, below you will find my phase 1 feedback

### Github Accounts and Team(s)
- [ ] Project Team table includes all relevant stakeholders including project lead(s), developers and reviewers.
- [ ] Github handles are provided for all team members
- [x] Roles are provided for all team members listed
- [ ] Contact info is provided for each team member (in order of importance):
  - [ ] github handle
  - [ ] nanoHUB username
  - [ ] slack handle
  - [x] email  

**The project team currently only consists of the developer and project lead.  Please include any other relevant team members if possible and include at least github handles for all.**

### Project Goals and Mission Statement
This section should provide a concise and clear answer to the following questions:
- [x] Why are we building this tool?
- [x] What is the key benefit
- [ ] How does this project relate to existing tools and existing software?
- [ ] How does this project benefit the nano **manufacturing** community at large?
- [x] Who will use this software?

**If the primary users are "researchers", explain why this tool will benefit them and as compared to other exiting software applications, or platforms.  Are there other potential users?**
 
### License
- [x] A standard license file has been added named "LICENSE".  The recommended licenses are: MIT or apache2
- [ ] Copyrights notices reflect the appropriate parent institution for example: `Copyright 20xx University of Illinois`, or `Copyright 20xx The Regents of the University of California`

**The copyright notice should be change to `...University of Illinois`**

#### References
https://copyright.illinois.edu/  
https://copyright.universityofcalifornia.edu/  

### User Classes: Section 3.1
- [ ] The tasks the class of users will perform are considered.
- [ ] Access and privilege level (if relevant).
- [ ] Features used and frequency on interaction.
- [ ] Experience/knowledge level.
- [ ] Type of interaction (education, research, other…) and platform (nanoHUB, API, local use, etc.)

**No clear user classes are defined here and the tasks the users may perform are not considered.  Also I think that some level of knowledge is needed, no?  For instance does this tool expect a researchers level of expertise, or is it useful for students or newcomers?**  

### User/System Requirements
- [ ] Each user requirement is complete and unambiguous.
  - [ ] All information needs to understand the requirement and whom it affects.
  - [ ] System or user perspectives are okay but again should by unambiguous with clear phrasing such as "the system shall" or "user x shall".
- [ ] Requirements are correct: They accurately reflect a capability that will meet some stakeholder(s) needs.
- [ ] A clear value that the proposed capability provides to stakeholders the is communicated.
- [ ] User requirements make clear connections to user classes defined in section 3.1.

**These are not well defined user requirements,  A good way to work with these is via a issue template.  I have added the template for creating these as GitHub issues here: https://github.com/nanoMFG/nanodiam/issues/new?assignees=&labels=user&template=user-requirement.md&title=
Here is an example
In order to provide open access for researchers in nanodiamond growth to rapid Raman spectrum analysis we will implement (some specific feature).
Note that in the example above, the rest of this requirement should be specific, testable and unambiguous.  Refer to the criteria (checkboxes) above and comments in the user requirement issue template for more tips on specifying user requirements.**

### User Outreach
- [ ] The outreach plan is specific mentioning user classes or specific labs or courses.
- [ ] Does the plan have a timeline?
- [ ] Does the plan mention specific team members that will perform the outreach activities
- [ ] Does the plan describe how these outreach efforts will be used to improve the project outcomes.

**Please review the criteria above and see if you can improve the outreach plan.**

**Comment by dadamsncsa** [comment]
@elifleaf @tawfick @AagamShah97 @greghbauer: REMINDER: In order to complete this phase 1 review the closing date (12/3).  Please offer your feedback ASAP.  If there are particular features or requirements you would like to suggest, this a good time.  The formal rubric I used above is a good guide, but need not be used in its entirety by all reviewers.  Please feel free to offer your advice and suggestions on any or all particular areas during this RFC period. You can find the phase 1 review guide here:
https://github.com/nanoMFG/community/blob/master/governance/planning-review.md
Thanks!

**Review by elifleaf — COMMENTED** [review]
Development Team: 
- I would recommend adding Darren and possibly Aagam as well as developers. 

Overview and Major Planned Features: 
- Here it would be good to provide a little more detail, for example when you say 'upload files' I think you may wish to specify 'upload a Raman spectrum dataset' or something like this. 
- Add that the tool will be automatically capable of quantifying things that are important in characterizing the synthesized nanodiamond.  I myself don't know exactly what the metrics that you quantify would be, but you will have a better idea.  For example, in graphene we measure the G, G', and D peaks and that gives information about crystalline quality, number of layers, etc. 

Background and Strategic Fit: 
- You may wish to note here that we intend that this tool will become part of a larger suite of tools for analysis of nanodiamond, including synthesis database, other characterization approaches, etc etc 

User outreach plan: 
- My main suggestion here would be for the first item to specify who you mean by colleagues.  For example you could start internally, with members of your research group and ask them to try out the tool.  Then you could reach out to external colleagues - probably Lili can recommend some - as a second step.

File: rfcs/20201113-nanodiam-1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+## Development Team
+<!-- Complete table for all team members 
+ roles: lead, developer, reviewer
+ status: active, inactive
+-->
+Name | Role | github user | nanohub user | email | status
+---|---|---|---|---|---
+Lili Cai| Lead | | |lilicai@illinois.edu|
+Adrian Manjarrez | Developer | adrianm3 | amanjarrez | adrianm3@illinois.edu | active
+ |  |  |  |  |
```
Comment (dadamsncsa) [inline rfcs/20201113-nanodiam-1.0.0.md:17]: Here are the details for adding Aagam and I here as developers as per @elifleaf recommendation:
Darren Adams | developer | dadamsncsa | Darren K Adams | dadams@illinois.edu | active
Aagam Shah | developer | AagamShah97 |  aagam2 | aagam2@illinois.edu | active

**Comment by dadamsncsa** [comment]
@adrianm3 Are you able to address the recommendations above to finish phase 1 and close this pr?
@nanoMFG/nanodiam-dev Please add comments here, line comments in the "Files changed" tab or commit your own SPD updates to @adrianm3 's branch to help address the RFC recommendations and move on to phase 2.  Thanks!

**Review by dadamsncsa — APPROVED** [review]

**Pull request merged**